### DISCUSSION TITLE
Unhandled Runtime Error

### DISCUSSION THREAD

**Discussion by dhawansolanki** [RakeshPotnuru/Blog#5] [Q&A] [answered]
Error: Hydration failed because the initial UI does not match what was rendered on the server.

Call Stack
throwOnHydrationMismatch
node_modules\react-dom\cjs\react-dom.development.js (14344:0)
tryToClaimNextHydratableInstance
node_modules\react-dom\cjs\react-dom.development.js (14372:0)
updateHostComponent$1
node_modules\react-dom\cjs\react-dom.development.js (20636:0)
beginWork
node_modules\react-dom\cjs\react-dom.development.js (22373:0)
HTMLUnknownElement.callCallback
node_modules\react-dom\cjs\react-dom.development.js (4157:0)
Object.invokeGuardedCallbackDev
node_modules\react-dom\cjs\react-dom.development.js (4206:0)
invokeGuardedCallback
node_modules\react-dom\cjs\react-dom.development.js (4270:0)
beginWork$1
node_modules\react-dom\cjs\react-dom.development.js (27243:0)
performUnitOfWork
node_modules\react-dom\cjs\react-dom.development.js (26392:0)
workLoopSync
node_modules\react-dom\cjs\react-dom.development.js (26303:0)
renderRootSync
node_modules\react-dom\cjs\react-dom.development.js (26271:0)
performConcurrentWorkOnRoot
node_modules\react-dom\cjs\react-dom.development.js (25577:0)
workLoop
node_modules\scheduler\cjs\scheduler.development.js (266:0)
flushWork
node_modules\scheduler\cjs\scheduler.development.js (239:0)
MessagePort.performWorkUntilDeadline
node_modules\scheduler\cjs\scheduler.development.js (533:0)

**Comment by RakeshPotnuru** [comment, marked as answer]
Hi @dhawansolanki 
Please refer to the discussion [here](https://github.com/vercel/next.js/discussions/35773).
